app/templates/app/_build.gradle

```diff
@@ -14,17 +14,19 @@ repositories {
 /*
  * App compilation configuration
  */
- 
-apply plugin: 'android-sdk-manager'
+
 apply plugin: 'com.android.application'
+
 android {
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
 
     defaultConfig {
-        minSdkVersion <%= androidMinSdkVersion %>
-        targetSdkVersion <%= androidTargetSdkVersion %>
+        minSdkVersion 14
+        targetSdkVersion 21
```

Comment:
Should we also remove where we prompt the user what minSdkVersion and TargetSdkVersion?